test/utils/deployment.ts

```diff
@@ -111,14 +110,16 @@ export async function setupDependenciesAsync(logType: LogType = LogType.Hidden):
  * Tears down 0x-api's dependencies.
  */
 export async function teardownDependenciesAsync(): Promise<void> {
-    const down = spawn('docker-compose', ['down'], {
-        cwd: apiRootDir,
-    });
-    await new Promise<void>(resolve => {
-        down.on('close', () => {
-            resolve();
-        });
-    });
+    await waitForCloseAsync(
+        spawn('docker-compose', ['down'], {
+            cwd: apiRootDir,
+        }),
+    );
+    await waitForCloseAsync(
+        spawn('docker-compose', ['rm', '-v'], {
```

Comment:
could be timing out if it's prompting for confirmation? use `-f` flag to override